CourseGrab/Home/HomeTableViewHeader.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  HomeTableViewHeader.swift
+//  CourseGrab
+//
+//  Created by Mathew Scullin on 2/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class HomeTableViewHeader: UITableViewHeaderFooterView {
+
+    private let title = UILabel()
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+
+        contentView.backgroundColor = .white
+        title.font = ._20Semibold
+        contentView.addSubview(title)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setupContraints (section: Int) {
+        let padding = section == 0 ? 24 : 6
+        title.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(20)
+            make.bottom.equalToSuperview().inset(6)
+            make.top.equalToSuperview().inset(padding)
+        }
+    }
+
+    func configure(section: Int, numSections: Int, count: Int, totalCount: Int) {
```

Comment:
I have an alternative idea, but maybe it can wait for another PR since it effects other code too. I'll put it here anyway so I don't forget and see what you all think:

Define this enum in `HomeViewController`:
```
enum TableSection {
    case available([Section]), awaiting([Section])
}
```

Then, `HomeTableViewHeader.configure` takes a `TableSection` and doesn't do any math. No stinky subtraction or annoying ifs needed!

Reply:
i agree with this. this is also in our bible so you can reference that if you'd like (https://bible.cornellappdev.com/standards/ios/tableviews)